docs/README.md

```diff
@@ -5,33 +5,39 @@
 </p>
 
 # Monster Hunter Now Events
+## Table Of Contents
+* Calendar Subscription (Purpose)
+  - Steps to Subscribe Event Calendar <br>
+* Local Implementation of Calendar Subscription <br>
+* Tools <br>
+* References 
 
-This is a calendar subscription file for events in [Monster Hunter Now](https://monsterhunternow.com/).
+## Calendar Subscription
+This is a calendar subscription page for events in [Monster Hunter Now](https://monsterhunternow.com/).
 
-If you aren't playing yet, [feel free to join](https://apps.apple.com/us/app/monster-hunter-now/id6445906110) and use my referral code for free items and zenny, `89K5MKCF`.
+In case you are new to Events, [Feel free to explore and join the enriching experience.](https://apps.apple.com/us/app/monster-hunter-now/id6445906110). Use the below referral code for free items and zenny, *`89K5MKCF`*.
 
-I created this because I didn't want to add events to a shard calendar for my friends and I any longer and it's also a great way to test out how beneficial AI is in coding via dialoge with ChatGPT, and an excuse to utilize the OpenAI API.
+This has been created to add events to a shared calendar for all the players including me and my friends. It's a great way to visualize how AI is beneficial for coding via dialogue with ChatGPT, and an exemplary way of utilizing the OpenAI API. 
```

Comment:
While I appreciate the consolidation, it also loses the nuance of how I ended up creating this tool. The purpose was because I didn't want to maintain a manually created, shared calendar.